ssm/agents.py

```diff
@@ -329,11 +328,12 @@ def run_receiver(protocol, brokers, project, token, cp, log, dn_file):
                     if protocol == Ssm2.STOMP_MESSAGING:
                         ssm.send_ping()
 
-            except ( 
+            except (
                 NotConnectedException,
                 AmsConnectionException,
-                requests.Timeout,
-                requests.ConnectionError) as error:
+                TimeoutError,
+                ConnectionError
```

Comment:
Is python able to find these classes without the explicit reference to the requests module?

Reply:
The requests can handle timeout errors or connection(network) related errors. But, It seems we are not leveraging the features of it fully.

TimeoutError and ConnectionError are built-in standard libraries in Python 3. It seems VSCODE extensions came to the rescue when I used python2.7. 

<img width="618" alt="built-in libs" src="https://github.com/apel/ssm/assets/127398882/16d9a1b7-d72f-480f-ab8e-601ed94fe3bb">


 So. AMS offers an exception to handle timeouts by itself. It should be working fine now, with AMS API timeout exceptions dealt with by the lib.

Reply:
AMS has these exceptions to handle errors:

class AmsException(Exception):
class AmsServiceException(AmsException):
class AmsBalancerException(AmsServiceException):
class AmsTimeoutException(AmsServiceException):
class AmsConnectionException(AmsException):
class AmsMessageException(AmsException):

Reply:
@gregcorbett , @tofu-rocketry .. Just wanted to let you guys know that I have updated the description.